react-client/src/components/pages/register/service.ts

```diff
@@ -0,0 +1,22 @@
+import axios from "axios"
+
+
+export type userType = {
+
+    id?: number;
+    first_name: string;
+    last_name: string;
+    email: string;
+    password: string;
+    role?: string;
+  
+  }
+  
+const BASE_URL = `http://localhost:3000`
```

Comment:
in the MR i see you repeating the localhost:300 over 10 times! 
too bad, you can centerlised it to single settings